Library/Homebrew/formula_installer.rb

```diff
@@ -1256,7 +1256,9 @@ def downloader
 
   sig { void }
   def pour
-    if Homebrew::EnvConfig.verify_attestations? && formula.tap&.core_tap?
+    # We skip `gh` to avoid a bootstrapping cycle, in the off-chance a user attempts
+    # to explicitly `brew install gh` without already having a version for bootstrapping.
+    if Homebrew::EnvConfig.verify_attestations? && formula.tap&.core_tap? && formula.name != "gh"
```

Comment:
Could probably do something like this?
```suggestion
    if Homebrew::EnvConfig.verify_attestations? && formula.tap&.core_tap? && Formula["gh"].any_version_installed?
```

Reply:
Is it possible that someone has a really old version of `gh` pinned and that doesn't support attestations? That's probably a super rare edge-case though.

Reply:
Yeah, older versions won't necessarily work, so the more general check here would include a minimum version. But that would also preclude non-formula `gh` installs without some extra logic, so personally inclined to let this edge case fall through since it's already an edge case within an edge case 🙂